### DISCUSSION TITLE
Get Join

### DISCUSSION THREAD

**Discussion by ptids** [katorlys/HelpCenter#4] [Join Request]
### Profile Information
**Github username:ptids
**Contact email: 

### What requirements you meet

- Write plugins to prove my own value

### Your thoughts

- I'm joining Katorly Lab because  A developer of Minecraft 
- I'm going to  coding

**Comment by katorly** [comment]
Please fill in all the blanks

**Reply by ptids** [reply]
抱歉，现已填写所有内容.

**Reply by katorly** [reply]
> 抱歉，现已填写所有内容.

没看到你填了啥呀……你那里github有没有出问题呀

**Reply by ptids** [reply]
没有问题呀

**Comment by ptids** [comment]
![捕获](https://github.com/katorlys/HelpCenter/assets/132898007/a4f0e6a4-dd99-44c0-990f-ef595f8cfd3f)

**Reply by katorly** [reply]
其它信息呢……  
那些“帮你填好”的也是需要你填写的，你可以参考一下 #3

**Reply by ptids** [reply]
我填了，忘记把<--XX-->去掉了

**Comment by katorly** [comment]
Welcome!  
记得加入 Gitter 聊天室（可用 Github 登录），方便交流哦~ 👉https://app.gitter.im/#/room/#katorlys:gitter.im
